src/app/components/Notifications/Notifications.contracts.ts

```diff
@@ -24,11 +24,32 @@ export interface NotificationTransactionItemProperties {
 	containmentRef?: any;
 	onVisibilityChange?: (isVisible: boolean) => void;
 	onTransactionClick?: (item?: DTO.ExtendedConfirmedTransactionData) => void;
-	isCompact: boolean;
 }
 
 export interface NotificationsProperties {
 	profile: Contracts.IProfile;
 	onNotificationAction?: (id: string) => void;
 	onTransactionClick?: (item?: DTO.ExtendedConfirmedTransactionData) => void;
 }
+
+export interface NotificationsMigrationItemProperties {
+	// @TODO: assign a proper type for this once defined
```

Comment:
@alfonsobries if I am not mistaken, I think the type here is `DTO.ExtendedConfirmedTransactionData` :hmhm:

Reply:
@goga-m `ExtendedConfirmedTransactionData` contains some equivalences from most of the properties, but it uses methods instead of properties like the fake data I'm using but does not have the polygon `migrationAddress`, The transactions table is also using similar sample data 

I predict that once we save/retrieve the transactions, we will have to define or adapt one of those transaction types both here and in the transaction table. I think it will be a good time to define that type once that happens.